docker/clio_docker/centos/install_cmake.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
You can use the `/usr/bin/env` trick here too or just use `/bin/bash` everywhere but I'd rather not mix the two